src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public static final int FIGURE_TYPES = 5;
+    public static final int SIDE_MAX_VALUE = 100;
+    private static final int DEFAULT_CIRCLE_RADIUS = 10;
+    private static final String DEFAULT_CIRCLE_COLOR = "WHITE";
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureCount = 1 + (int) (Math.random() * FIGURE_TYPES);
+        int firstSide = 1 + (int) (Math.random() * SIDE_MAX_VALUE);
+        int secondSide = 1 + (int) (Math.random() * SIDE_MAX_VALUE);
+        int height = 1 + (int) (Math.random() * SIDE_MAX_VALUE);
+
+        switch (figureCount) {
+            case 1: return new Rectangle(colorSupplier.getRandomColor(), firstSide, secondSide);
```

Comment:
You can move the creation of each shape into a separate method. For example getRandomRectangle();